hr_timesheet_role/views/res_config_settings.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <!--
+        Copyright 2018 Brainbean Apps (https://brainbeanapps.com)
+        License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+    -->
+
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="name">res.config.settings.view.form.hr_timesheet_timetracker</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="hr_timesheet.res_config_settings_view_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//div[@name='section_roles_and_assignments']" position="inside">
+                <div class="row o_settings_container">
+                    <div class="col-12 col-lg-6 o_setting_box">
+                        <div class="o_setting_left_pane">
+                            <field name="is_timesheet_role_required"/>
+                        </div>
+                        <div class="o_setting_right_pane">
+                            <label for="is_timesheet_role_required" string="Require role in Timesheets"/>
+                            <span class="fa fa-lg fa-building-o" title="Values set here are company-specific." role="img" aria-label="Values set here are company-specific." groups="base.group_multi_company"/>
```

Comment:
@tarteo actually the icon should be there

Reply:
Seems only the snail mail module don't use the multi company group :S weird. I thought the group was active because it was shown on the snail mail configuration options.

Reply:
Lucky choice :)